test/functional/results_controller_test.rb

```diff
@@ -465,7 +465,11 @@ def teardown
 
           should 'and the result is available' do
             SubmissionFile.make(:submission => @submission)
-            Mark.make(:result => @result)
+            if @assignment.marking_scheme_type == Assignment::MARKING_SCHEME_TYPE[:rubric]
+              Mark.make(:rubric, :result => @result)
```

Comment:
I changed it. Thanks. I noticed many other instances in this file in which we use the old syntax. Is this something I should change? Is it worth making an issue for?